site/docs/reference/Connectors/materialization-connectors/google-gcs-csv.md

```diff
@@ -0,0 +1,92 @@
+
+# CSV Files in Google GCS
+
+This connector materializes [delta updates](../../../concepts/materialization.md#delta-updates) of
+Flow collections into a GCS bucket in the CSV format.
+
+The delta updates are batched within Flow, converted to CSV files, and then pushed to the GCS bucket
+at a time interval that you set.
+
+[`ghcr.io/estuary/materialize-gcs-csv:dev`](https://ghcr.io/estuary/materialize-gcs-csv:dev)
+provides the latest connector image. You can also follow the link in your browser to see past image
+versions.
+
+## Prerequisites
+
+To use this connector, you'll need:
+
+* A GCS bucket to write files to. See [this
+  guide](https://cloud.google.com/storage/docs/creating-buckets) for instructions on setting up a
+  new GCS bucket.
+* A Google Cloud [service account](https://cloud.google.com/docs/authentication/getting-started)
+  with [`roles/storage.objectAdmin`](https://cloud.google.com/storage/docs/access-control/iam-roles#standard-roles)
```

Comment:
Could we recommend less privileged roles?

Reply:
Updated this to only require the `storage.objectCreator` role, which should be enough for the materialization to GCS.

Reply:
Thanks!